packages/pilot/src/pages/Recipients/Detail/index.js

```diff
@@ -48,7 +49,12 @@ const mapStateToProps = (state = {}) => {
     },
```

Comment:
Ao fazer o login nós guardamos o recebedor principal no redux, pra facilitar pode puxar de lá (a vantagem é não precisar lidar com environment)

```suggestion
    },
    defaultRecipient: {
      id: defaultRecipientId,
    }
```
Assim basta remover o `default_recipient_id` de company